cli/src/pcluster/api/controllers/cluster_operations_controller.py

```diff
@@ -353,7 +353,6 @@ def update_cluster(
                 force=force_update,
                 validator_suppressors=get_validator_suppressors(suppress_validators),
                 validation_failure_level=FailureLevel[validation_failure_level],
-                dry_run=dryrun,
```

Comment:
We are removing this parameter because there is no code using the dryrun variable anymore into the `validate_create_request` and `validate_update_request` methods.

The dryrun variable was used only for a "scheduler-plugin" specific branch of the code that has been removed as part of this patch.

See `validate_create_request` and `validate_update_request` changes in this patch.

We can improve the code if we want in future to execute a more precise dryrun in the validate methods but as part of this patch I'm just removing scheduler-plugin specific code, I don't want to change internal logic.